app/__init__.py

```diff
@@ -10,17 +10,18 @@
 db = SQLAlchemy()
 login_manager = LoginManager()
 
+
```

Comment:
Most of the changes in this file are formatting, only lines 52 and 54 are really new.